src/components/ReferralCard.astro

```diff
@@ -0,0 +1,30 @@
+---
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "./ui/card";
+import { Button } from "./ui/button";
+
+interface Props {
+	url: string;
```

Comment:
incorrect indentation
```suggestion
    url: string;
```

Reply:
could probably do with going through prettier. i just now saw that there is a astro plugin: https://github.com/withastro/prettier-plugin-astro